src/main/java/TaskList.java

```diff
@@ -0,0 +1,2 @@
+public class TaskList {
+}
```

Comment:
Perhaps it would be better to shift the `List<String> taskList` from `Jarvis.java` to here for better abstraction, allowing you to define methods with cleaner code.